### ISSUE TITLE
About regularization of entity embedding and relation embedding

### ISSUE THREAD

**Issue by RichardHGL** [thunlp/TensorFlow-TransX#18]
I also have a try on Fast-TransX, which do regularization on entity embedding and relation embedding. Tensorflow-TransE's result is much worse than Fast-TransE, maybe it make sense. So Why not put norm constraint on entity embedding and relation embedding on Tensorflow-TransX.

**Comment by THUCSTHanxu13** [comment]
We have done some experiments before. And the experimental results show that the influence of the norm constraint is not very notable. Hence, we do not put norm constraint on embeddings for acceleration. I guess that you run our code in Tensorflow-TransX directly. If you set the dimension with 100, and training epochs with 1000. Tensorflow-TransX should achieve comparable results compared to Fast-TransX.

Thank you for your suggestion ~ We will add norm components for our project.

**Comment by RichardHGL** [comment]
Thanks for your reply, I'll have a try with dimension 100.

**Comment by THUCSTHanxu13** [comment]
I have add some essential normalization for our code.
